README.md

```diff
@@ -60,6 +60,11 @@ Amplication Data Service Generator is the component that generates the server-si
 
 To use this package as a library or as a CLI with other projects, follow the instructions on the package page.
 
+### [Amplication Scheduler](./packages/scheduler/README.md)
```

Comment:
Should we really list all the packages in the README? isn't that takes the focus from the main target?

Reply:
I think it's okay at the bottom of the document so people would know what do we hold in the monorepo